Wordle.Api/Wordle.Api/Controllers/PersonController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using System.Numerics;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("Player")]
+public class PlayerController : ControllerBase
```

Comment:
Your file name is `PersonController.cs` but your class name is `PlayerController`, this is causing conflicts with your other file, called `PlayerController.cs`

![image](https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/assets/43832670/009fbe1d-6518-4163-b175-c45cecb959b1)